boards/stm32f3discovery/Makefile.include

```diff
@@ -2,6 +2,9 @@
 export CPU = stm32f3
 export CPU_MODEL = stm32f303vc
 
+# use on-board LEDs
+USEMODULE += onboard_led
```

Comment:
true. I guess it does indeed make sense to only pull the module if needed.

Further thought: we could also move the call to `onboard_led_init` from the board initialization into the cpu arch initialization, e.g. the `reset_handler_default` for all cortexm based boards...